src/embed/main.js

```diff
@@ -79,19 +87,39 @@ function onVideoTap() {
   worldRenderer.videoProxy.play();
   hidePlayButton();
 
+  // TODO(hoch): refactor this.
+  omnitoneDecoder.initialize().then(function () {
+    console.log('initialized');
+    videoElement.play();
+  }, function (error) {
+    console.log(error);
+  });
+
   // Prevent multiple play() calls on the video element.
   document.body.removeEventListener('touchend', onVideoTap);
 }
 
 function onRenderLoad(event) {
   if (event.videoElement) {
+
+    // TODO: activate this only when it's in the 'spatial' audio mode.
+    audioContext = new AudioContext();
+    omnitoneDecoder = Omnitone.createFOADecoder(
+        audioContext, event.videoElement, { postGainDB: 10 });
+
     // On mobile, tell the user they need to tap to start. Otherwise, autoplay.
     if (Util.isMobile()) {
       // Tell user to tap to start.
       showPlayButton();
       document.body.addEventListener('touchend', onVideoTap);
     } else {
-      event.videoElement.play();
+      // TODO(honch): refactor this.
```

Comment:
Probably pull this and the initialize() from the other if branch into its own method. initOmnitone or something like that?